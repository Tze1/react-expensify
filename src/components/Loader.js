import React from 'react';

const Loader = () => (
  <div className="loader component">
    <img src="/img/loader.gif" title="loading..." />
  </div>
);

export default Loader;
