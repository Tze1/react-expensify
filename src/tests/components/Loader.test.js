import React from 'react';
import { shallow } from 'enzyme';
import Loader from '../../components/Loader';

describe('Loader component', () => {
	it('Should render Loader properly', () => {
	  const wrapper = shallow(<Loader />);

	  expect(wrapper).toMatchSnapshot();
	});
});
