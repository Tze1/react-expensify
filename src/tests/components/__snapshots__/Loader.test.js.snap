// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader component Should render Loader properly 1`] = `
<div
  className="loader component"
>
  <img
    src="/img/loader.gif"
    title="loading..."
  />
</div>
`;
